wdfn-server/waterdata/templates/iv-data-availability-statement.html

```diff
@@ -0,0 +1,61 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <div id="iv-data-availability-statement" class="usa-prose">
+        <h1>USGS Water Data for the Nation</h1>
```

Comment:
Can we remove this and put "Current and Historical<br>Instantaneous Data Availability" in the h1 tag. Then the h3 tags can change into h2 tags. This would follow the pattern of the Provisional Data Statement page.

Reply:
Great idea. Will do